src/utils/formatDate.ts

```diff
@@ -1,7 +1,7 @@
-const dateTimeFormatLocale = 'en-GB'
+import { getCurrentLanguage } from './currentLanguage'
 
 export function formatDateTime(date: Date) {
-  return new Intl.DateTimeFormat(dateTimeFormatLocale, {
+  return new Intl.DateTimeFormat(getCurrentLanguage(), {
```

Comment:
Please keep in mind that the site language can be given including a region: `Obj.root().update({_language: "en-gb"})`
While we could guess the visitor's language by looking at `navigator.languages` by prefix, this may
* create hydration issues for prerendered pages (which we don't yet support though)
* have unintended consequences if editors rely on WYSIWYG

Personally I don't see why Apr 29, 2024 would be a problem.
If we'd really like to go with a GB region I'd suggest to update the site language of the initial content.

Generally we should also make sure that the content language matches whatever we choose.

Reply:
Since the month is spelled out like "Apr" this is good enough. Thanks!